src/macaron/slsa_analyzer/specs/ci_spec.py

```diff
@@ -36,3 +37,6 @@ class CIInfo(TypedDict):
 
     provenances: Sequence[DownloadedProvenanceData]
     """The provenances data."""
+
+    build_info_results: InTotoV01Payload
+    """The build information results computed for a build step. We use the in-toto 0.1 as the spec."""
```

Comment:
It's mentioned in the PR description that `CIInfo["provenances"] is also renamed to CIInfo["build_info_results"]`. I think we should remove this `provenances` attribute here too

Reply:
My PR description was not accurate and I fixed it: Instead of using ``CIInfo["provenances"]`` for inferred build command analysis results, we use a new field: ``CIInfo["build_info_results"]``. `CIInfo["provenances"]` is still used for storing provenances.